railties/test/application/test_runner_test.rb

```diff
@@ -895,6 +895,20 @@ class DummyTest < ApplicationSystemTestCase
       assert_match "1 runs, 1 assertions, 0 failures, 0 errors, 0 skips", output
     end
 
+    def test_can_exclude_files_from_being_tested_via_default_rails_command_by_setting_DEFAULT_TEST_EXCLUDE_env_var
+      create_test_file "smoke", "smoke_foo"
+
+      output = Dir.chdir(app_path) { `DEFAULT_TEST_EXCLUDE="test/smoke/**/*_test.rb" bin/rails test` }
```

Comment:
Can we use `run_test_command` wrapped in `switch_env`?

Same as what we do here: https://github.com/rails/rails/blob/f4538aa5865da5f16b707d34aa733a5e2353f2fd/railties/test/commands/secrets_test.rb#L64-L68

Reply:
I looked for something like `switch_env` but couldn't find it, thanks 👍

What I liked about the previous approach is that both tests (for `rake test` and `rails t`) were symmetrical. But this is more consistent with other similar tests.